content/docs/tasks/security/authn-policy/index.md

```diff
@@ -1,90 +1,85 @@
 ---
```

Comment:
Please format the doc to no more than 160 chars per line.

Reply:
I'm not aware of a standard markdown reformatting tool. I usually just zip through with my text editor and add linefeeds so that everything fits.